data/sound.appdata.xml.in

```diff
@@ -7,6 +7,19 @@
   <summary>Adjust speaker and microphone volume or see Now Playing information and media controls</summary>
   <icon type="stock">preferences-desktop-sound</icon>
   <releases>
+    <release version="6.0.0" date="2021-07-14" urgency="medium">
+      <description>
+        <p>New features</p>
+          <li>select specific input and output devices</li>
```

Comment:
This looks like it's invalid HTML / XML, since the <li> tag is outside an <ul> / <ol> ?
At least the appdata validator complains about it.

Reply:
Yup, wrapping it in the missing `<ul>` tag fixes validation:
https://src.fedoraproject.org/rpms/wingpanel-indicator-sound/blob/rawhide/f/00-fix-appdata-xml-syntax.patch

Reply:
Can you open a new issue report please? A comment on a merged branch will probably get lost